ui/packages/docs/docs/reference/characters/travelerhydro.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Traveler (Hydro)
+---
+
+import HitlagTable from "@site/src/components/Hitlag/HitlagTable";
+import FieldsTable from "@site/src/components/Fields/FieldsTable";
+import ParamsTable from "@site/src/components/Params/ParamsTable";
+import FramesTable from "@site/src/components/Frames/FramesTable";
+import IssuesTable from "@site/src/components/Issues/IssuesTable";
+import AoETable from "@site/src/components/AoE/AoETable";
+import NamesList from "@site/src/components/Names/NamesList";
+import ActionsTable from "@site/src/components/Actions/ActionsTable";
+
+## Frames
+
+<FramesTable item_key="travelerhydro" />
+
+## Hitlag Data
+
+<HitlagTable item_key="travelerhydro" />
+
+## AoE Data
+
+<AoETable item_key="travelerhydro" />
```

Comment:
Missing AoE data. Copy paste normal, charge AoE from another traveler variant and then add the aoe for the other abilities using the following names: E-TorrentSurge, E-Aligned, E-Dewdrop, Q.